GeneratedMonitoringArtifacts/DefaultRecordingRulesList.txt

```diff
@@ -27,4 +27,36 @@ instance_device:node_disk_io_time_weighted_seconds:rate5m
 node_namespace_pod_container:container_memory_working_set_bytes
 node_namespace_pod_container:container_memory_rss
 node_namespace_pod_container:container_memory_cache
-node_namespace_pod_container:container_memory_swap
\ No newline at end of file
+node_namespace_pod_container:container_memory_swap
```

Comment:
just curious, do all the metrics in the list flow from cluster currently?

Reply:
No, at least the following metrics are not flowing :  https://github.com/prometheus-community/windows_exporter/issues/1129